auction-app-frontend/src/App.jsx

```diff
@@ -1,11 +1,46 @@
-import { TestPage } from './pages'
+import { Routes, Route } from "react-router-dom";
+
+import { Navbar, Header, Footer, Breadcrumbs } from "src/components";
+import {
+  AboutUs,
+  PrivacyPolicy,
+  TermsAndConditions,
+  Home,
+  Shop,
+  MyAccount,
+} from "src/pages";
+
+import { BreadcrumbProvider } from "src/BreadcrumbContext";
```

Comment:
Create folder for context related stuff.
U can create for example `store` folder and store it there.